backend/clubs/views.py

```diff
@@ -4861,6 +4863,39 @@ def get_queryset(self):
         )
 
 
+class WhartonCyclesView(viewsets.ModelViewSet):
+    """
+    update: Update application cycle
+    list: Get list of all application cycles
+    """
+
+    permission_classes = [WhartonApplicationPermission | IsSuperuser]
+    http_method_names = ["get", "post", "put", "patch", "delete"]
+    serializer_class = ApplicationCycleSerializer
+
+    def get_queryset(self):
+        return ApplicationCycle.objects.all().order_by("end_date")
+
+    def list(self, request, *args, **kwargs):
```

Comment:
Not 100% but don't think you need this, list should exist already and will have exactly the behaviour of calling the super class' method

Reply:
Tested and it works without, thanks!